swap/src/protocol/alice/message2.rs

```diff
@@ -79,7 +79,7 @@ impl NetworkBehaviourEventProcess<RequestResponseEvent<BobToAlice, AliceToBob>>
                 if let BobToAlice::Message2(msg) = request {
```

Comment:
Could avoid a level of nesting by including this pattern in the outer match. Not sure if it worth it though.